src/accelerate/checkpointing.py

```diff
@@ -240,6 +249,7 @@ def load_accelerator_state(
     # Random states
     try:
         states = torch.load(input_dir.joinpath(f"{RNG_STATE_NAME}_{process_index}.pkl"))
+        override_attributes["step"] = states["step"]
```

Comment:
This is a breaking change for people upgrading accelerate.  If previously saved pickled states objects don't contain a "step" key, then random states fail to initialize correctly when you resume a training that was checkpointed from a prior version of accelerate.

Reply:
Correct, we've fixed this in a patch release

Reply:
Ah I see [this](https://github.com/huggingface/accelerate/pull/2992) now.  Thanks!